server/src/lib.rs

```diff
@@ -1,3 +1,13 @@
+#[cfg(all(target_env = "msvc", feature = "jemalloc"))]
+compile_error!("'jemalloc' feature cannot be used when MSVC is being used");
+
+#[cfg(all(not(target_env = "msvc"), feature = "jemalloc"))]
+use tikv_jemallocator::Jemalloc;
+
+#[cfg(all(not(target_env = "msvc"), feature = "jemalloc"))]
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
```

Comment:
shouldn't this whole block be moved to `main.rs` of server?

AFAIK `lib.rs` is used only for integration tests - I might be wrong tho :D

Reply:
I added it to the `lib.rs` file instead of `main.rs` if the server would ever become a separate library. There might come a use case for it where you'd want to swap async runtimes or something. Happy to move it to `main.rs` if there is no point to keep it in `lib.rs`.